meta/main.yaml

```diff
@@ -0,0 +1,13 @@
+---
```

Comment:
A `meta/main.yml` file is required to install the role using ansible-galaxy. See [the docs](http://docs.ansible.com/ansible/latest/galaxy.html#installing-multiple-roles-from-a-file) for more info.